source/change-streams/change-streams.rst

```diff
@@ -417,6 +417,10 @@ Driver API
      * Server versions prior to 4.4 only support string as comment,
      * and providing a non-string type will result in a server-side error.
      *
+     * If a comment is provided, drivers MUST attach this comment to all
+     * subsequent getMore commands run on the same cursor.id returned from
+     * the aggregate command.
```

Comment:
I don't think we refer to `cursor.id` anywhere else like this, but this gets the same point across with less words.

```suggestion
     * subsequent getMore commands run on the same cursor.
```

Feel free to accept/reject. I'll suggest on the other lines for your convenience in case you want to apply them as a batch.

Reply:
Sounds much better, I agree. Thank you!